arch/arm/configs/miyoo_defconfig

```diff
@@ -1,14 +1,19 @@
 CONFIG_LOCALVERSION="-miyoo"
 # CONFIG_LOCALVERSION_AUTO is not set
 CONFIG_DEFAULT_HOSTNAME="miyoo"
-CONFIG_CGROUPS=y
+CONFIG_LOG_BUF_SHIFT=16
+CONFIG_PRINTK_SAFE_LOG_BUF_SHIFT=12
+CONFIG_CC_OPTIMIZE_FOR_SIZE=y
+# CONFIG_MULTIUSER is not set
+# CONFIG_KALLSYMS is not set
+CONFIG_EMBEDDED=y
```

Comment:
what stuff is being cut with this opt enabled? I couldn't read much from descrip. and in src

Reply:
This allow to configure expert parameters. In this case I removed multiuser, as we always use only root